### PR TITLE
docs: add release notes section and starting content

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

File: docs/api-reference/prod/info/post-v2infobalance.mdx
Comment (NotJeremyLiu) [inline docs/api-reference/prod/info/post-v2infobalance.mdx:1]: Why did we change the name of this file, isn't the endpoint called balances?

File: docs/general/getting-started.mdx [hunk 1]
```diff
@@ -39,14 +39,16 @@ Skip Go API includes several different endpoint types which allow developers to
...
- **Have questions or feedback? Help us get better!**
- 
- Join [our Discord](https://skip.build/discord) and select the "Skip Go Developer" role to share your questions and feedback.
-</Info>
\ No newline at end of file
+There are 3 ways to leverage Skip Go API to build cross-chain swapping & transferring experiences, depending on whether you're optimizing for control or for integration speed.
+
+<CardGroup cols={3}>
+  <Card title="Rest Endpoint — Most control, slowest" icon="sliders" href="/general/overview-and-typical-usage">
+    Use REST endpoints for low-level control over your interactions.
```
Comment (NotJeremyLiu) [inline docs/general/getting-started.mdx:46]: This extra organization is fire, we should make a note/linear ticket to turn this api one into a "Getting Started" page like the others have imo, wdyt?

Comment (NotJeremyLiu) [reply docs/general/getting-started.mdx:46]: nit: wdyt about adding "integration" at the end of each, so would change "Most control, slowest" to "Most control, slowest integration"

File: docs/general/getting-started.mdx [hunk 2]
```diff
@@ -39,14 +39,16 @@ Skip Go API includes several different endpoint types which allow developers to
...
+  </Card>
+  <Card title="Client Library — High control, fast" icon="code" href="/client/getting-started" >
+    Use the TypeScript library to abstract the complexity of making HTTP calls and access related helper functionality. 
+  </Card>
+  <Card title="Swap Widget — Medium control, fastest" icon="bolt-lightning" href="/widget/getting-started" >
```
Comment (NotJeremyLiu) [inline docs/general/getting-started.mdx:51]: nit: "Swap" Widget is old name, we should just call it Widget

File: docs/general/getting-started.mdx [hunk 3]
```diff
@@ -39,14 +39,16 @@ Skip Go API includes several different endpoint types which allow developers to
...
+    Embed the Skip Go SwapWidget in your frontend in a single line of code to launch with no developer effort.
```
Comment (NotJeremyLiu) [inline docs/general/getting-started.mdx:52]:
```suggestion
    Embed the Skip Go Widget in your frontend in a single line of code to launch with no developer effort.
```

File: docs/mint.json [hunk 4]
```diff
@@ -56,7 +67,7 @@
       ]
     },
     {
-      "group": "Skip Go Widget",
+      "group": "Skip Go Swap Widget",
```
Comment (NotJeremyLiu) [inline docs/mint.json:70]: nit: I think we should keep it just Widget

File: docs/release-notes/api.mdx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: "REST API"
+description: "Selected updates and improvements to the Skip Go REST API."
+---
+
+## September 28, 2024
+- Speed and performance improvements for `/balances` endpoint
+
+## August 21, 2024
+- Add `/balances` endpoint
```
Comment (NotJeremyLiu) [inline docs/release-notes/api.mdx:10]: Would be nice to link the api reference for endpoint announcements

File: docs/release-notes/overview.mdx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Stay up to date with Skip Go's latest features and improvements.
+
+<CardGroup cols={3}>
+
+  <Card title="Client Library" icon="code" href="/release-notes/client" >
+    A library that streamlines interaction with the Skip Go API, enabling cross-chain swaps and transfers across multiple ecosystems
+  </Card>
+  <Card title="Swap Widget" icon="bolt-lightning" href="/release-notes/swap-widget" >
```
Comment (NotJeremyLiu) [inline docs/release-notes/overview.mdx:8]:
```suggestion
  <Card title="Widget" icon="bolt-lightning" href="/release-notes/swap-widget" >
```

File: docs/api-reference/prod/info/post-v2infobalance.mdx
Comment (ericHgorski) [reply docs/api-reference/prod/info/post-v2infobalance.mdx:1]: temporary confusion on my part. resolved, good catch!

File: docs/general/getting-started.mdx (hunk 1 above)
Comment (ericHgorski) [reply docs/general/getting-started.mdx:46]: done and done!

**Review by NotJeremyLiu — APPROVED** [review]
fire

**Pull request merged**